src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -6077,19 +6084,26 @@ checkNullable:
             Return ReportFeatureUnavailable(feature, node, _scanner.Options.LanguageVersion)
         End Function
 
-        Friend Function CheckFeatureAvailability(feature As Feature) As Boolean
-            Return CheckFeatureAvailability(_scanner.Options.LanguageVersion, feature)
+        Friend Shared Function CheckFeatures(feature As Feature, opts As VisualBasicParseOptions) As Boolean
```

Comment:
Note: This checks for the presence of the feature flags, supplied via command line options.